tests/integration/test_api.py

```diff
@@ -32,6 +34,89 @@
     },
 ]
 
+nasa_statuses = {
+    PROD: {
+        "success": True,
+        "statuses": [
+            {"name": "MMT (SIT)", "status": "OK"},
+            {"name": "Status App Test Application", "status": "OK"},
+            {"name": "NSIDC DAAC OpenAltimetry", "status": "OK"},
+            {"name": "ED Pub", "status": "OK"},
+            {"name": "Earthdata Login", "status": "OK"},
+            {"name": "Common Metadata Repository (PROD)", "status": "OK"},
+        ],
+    },
+    UAT: {
+        "success": True,
+        "statuses": [
+            {"name": "MMT (SIT)", "status": "OK"},
+            {"name": "Status App Test Application", "status": "OK"},
+            {"name": "NSIDC DAAC OpenAltimetry", "status": "OK"},
+            {"name": "ED Pub", "status": "OK"},
+            {"name": "Earthdata Login (UAT)", "status": "OK"},
+            {"name": "Common Metadata Repository", "status": "OK"},
+        ],
+    },
+}
+
+
+@pytest.mark.parametrize("system, kwargs", [[PROD, UAT], nasa_statuses])
+@responses.activate
+def test_earthdata_status_happy_path(system, kwargs):
+    statuses = {
+        service["name"].replace(" (PROD)", "").replace(" (UAT)", "").strip(): service[
+            "status"
+        ]
+        for service in nasa_statuses[system]["statuses"][-2:]
+    }
+
+    responses.add(
+        responses.GET,
+        system.status_api_url,
+        json=nasa_statuses[system],
+        status=200,
+    )
+
+    result = earthaccess.status(system)
+    assert isinstance(result, dict)
+    assert result == statuses
+
+
+@pytest.mark.parametrize("system", [PROD, UAT])
```

Comment:
We don't need to parametrize this one as there is no difference in the response data between prod and UAT!

```suggestion
```